Sources/Datadog/DatadogCore/Context/NetworkConnectionInfoPublisher.swift

```diff
@@ -21,12 +21,19 @@ import Network
 /// This adds the necessary thread-safety and keeps the convenience of pulling.
 @available(iOS 12, tvOS 12, *)
 internal struct NWPathMonitorPublisher: ContextValuePublisher {
+    private static let defaultQueue = DispatchQueue(
+        label: "com.datadoghq.nw-path-monitor-publisher",
+        target: .global(qos: .utility)
+    )
+
     let initialValue: NetworkConnectionInfo
 
     private let monitor: NWPathMonitor
     private let queue: DispatchQueue
 
-    init(monitor: NWPathMonitor, queue: DispatchQueue) {
+    init(
+        monitor: NWPathMonitor = .init(),
+        queue: DispatchQueue = NWPathMonitorPublisher.defaultQueue) {
```

Comment:
nit/ weird it didn't raise linter issue:
```suggestion
        queue: DispatchQueue = NWPathMonitorPublisher.defaultQueue
    ) {
```